### PR TITLE
Update topic.html

### PR DESCRIPTION
**Pull request by qloakongithub**
### Resolves

This resolves the issue of accidentally closing a topic while trying to submit your post.

- Resolves the issue of accidentally closing a topic while trying to submit your post.

### Proposed Changes

It moves the Close Topic/Open Topic (for mods) to the top, beside Follow Discussion.

### Reason for Changes

Because, while making a topic, you may accidentally press Close Topic.

### CODE REVIEW

**Comment by PoIygon** [comment]
I remember someone saying that this repo isn’t the forum code they currently have right now or something

I really hope this gets accepted

File: djangobb_forum/templates/djangobb_forum/topic.html [hunk 1]
```diff
@@ -17,6 +17,14 @@
             <p class="error" style="display: none;">{% trans "There was an error :( Try again?" %}</p>
             <a {% if subscribed %}style="display: none;"{% endif %} class="follow-button button blue" href="{% url djangobb:forum_add_subscription topic.id %}" title="{% trans "Follow this discussion to get a message when there's a new post" %}">{% trans 'Follow Discussion' %}</a>
             <a {% if not subscribed %}style="display: none;"{% endif %} class="unfollow-button button grey" href="{% url djangobb:forum_delete_subscription topic.id %}" title="{% trans 'Stop getting messages about new posts to this discussion' %}">{% trans 'Unfollow Discussion' %}</a>
+	     {% if topic.closed %}
```
Comment (cobaltt7) [inline djangobb_forum/templates/djangobb_forum/topic.html:20]: `{% elif can_close %}` got lost

**Comment by Dhuls-Scratch** [comment]
[README.md](https://github.com/LLK/s2forums#this-project-is-no-longer-actively-maintained-by-the-scratch-team-for-a-list-of-active-open-source-projects-please-visit-httpsscratchmitedudevelopers)
> This project is no longer actively maintained by the Scratch Team. For a list of active open source projects please visit: https://scratch.mit.edu/developers

No pull request has been merged since #1338
In December **2015**

**Comment by benjiwheeler** [comment]
The forums code moved around 2016-17 to the scratchr2 repository, which is private.

We may consider this change at some point, but there isn't much engineering focus on the forums right now.

**Pull request closed without merging**